### ISSUE TITLE
Incorrect accounting of the CO2 emission costs in operating expenses ex ccs

### ISSUE THREAD

**Issue by KoenvanB** [quintel/etengine#1381]
The method `operating_expenses_excluding_ccs `seems to be lower than it should be, because the CO2 emission costs are unduly subtracted.

Operating expenses ex ccs are calculated by computing total operating expenses and then subtracting the ccs operating expenses
![image](https://github.com/quintel/etengine/assets/143791879/18d4ce16-a3fd-4027-a60a-e5771b65948c)

CCS operating expenses consist of the variable O&M costs for CCS plus CO2 emission costs
![image](https://github.com/quintel/etengine/assets/143791879/f8cb4710-d6f7-409b-a8ed-b1557cca1b52)

However, the method `variable_operation_and_maintenance_costs` only calculates the variable O&M costs based on the total costs per full load hour:
![image](https://github.com/quintel/etengine/assets/143791879/feeeaa87-137c-4bab-893f-d40a6d8e6cee)

![image](https://github.com/quintel/etengine/assets/143791879/4c3945fd-1351-4a9b-b674-fb6485e973f9)

In other words, `variable_operation_and_maintenance_costs` does **not** include CO2 emission costs.

As a result, `operating_expenses_ex_ccs` now consists of fixed and variable O&M costs ex ccs _minus_ the CO2 emission costs.

**Comment by mabijkerk** [comment]
@KoenvanB the `operating_expenses_excluding_ccs` subtracts not to the `operating_expenses_ccs`, but to the `variable_operation_and_maintenance_costs_for_ccs`:

```ruby
def operating_expenses_excluding_ccs
        fetch(:operating_expenses_excluding_ccs) do
          variable_operation_and_maintenance_costs -
            variable_operation_and_maintenance_costs_for_ccs +
            fixed_operation_and_maintenance_costs_per_year
        end
      end
```

The `variable_operation_and_maintenance_costs_for_ccs` do not seem to include CO2 emission costs:

```ruby
def variable_operation_and_maintenance_costs_for_ccs
        fetch(:variable_operation_and_maintenance_costs_for_ccs) do
          return 0.0 if input_capacity.zero?

          typical_input *
            variable_operation_and_maintenance_costs_for_ccs_per_full_load_hour /
            (input_capacity * 3600.0)
        end
      end
```

Am I missing something here?

**Comment by KoenvanB** [comment]
Nope, you are right, Mathijs. My bad, not a bug at all 😅 .
